RadixWallet/EngineKit/TXID.swift

```diff
@@ -3,10 +3,6 @@ import EngineToolkit
 public typealias TXID = TransactionHash
 
 extension TXID {
-	public func formatted(_ format: AddressFormat = .default) -> String {
```

Comment:
@GhenadieVP merge conflict imminent, but I see, it must happen, since you needed to implement this a method with this signature.